### PR TITLE
3.2 ui docs

### PR DESCRIPTION
**Pull request by SefiGiga**

### CODE REVIEW

File: source/guide/3.2/ui-login.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Login Screen
+After bootstrapping the manager with security support, the UI will redirect the users to the Login screen.<br>
+The standard `Username` and `Password` are required as well as the option to check `Remember me`. <br>
+
+![The Login Screen](/guide/images/ui/ui-login-screen.png)
+
+# Logout
+Once logged in, the user name is displayed on the top right of the screen, right next to the `Log out` button. <br>
+The user can opt to logout anytime by simply clicking on the `Log out`. <br>
```
Comment (nir0s) [inline source/guide/3.2/ui-login.md]: on the `Log Out` button (as shown below).

File: source/guide/3.2/ui-blueprint-upload.md [hunk 2]
```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
 ![The blueprint upload button](/guide/images/ui/ui_upload_blueprint_button.png)
 
 
-Once the blueprint upload dialog appears, the user can select a blueprint TAR file from the filesystem by pressing the `+` button:
+Clicking on it will cause the blueprint upload dialog to appear. <br>
+The user can either type in the path to the blueprint TAR file, or select it from the filesystem by pressing the `+` button:
```
Comment (nir0s) [inline source/guide/3.2/ui-blueprint-upload.md]: Is it only a TAR file? Didn't we add support for other formats?

Also, does the blueprint file have to be named "blueprint.yaml" ?

File: source/guide/3.2/ui-blueprint-upload.md [hunk 3]
```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
...
 
-![The blueprint upload dialog](/guide/images/ui/ui_upload_dialog.png)
+![The blueprint upload dialog](/guide/images/ui/ui-upload-blueprint.png)
 
+The blueprint name is a required field. The user can also provide a custom blueprint YAML file name. If none is provided, the default `blueprint.yaml` file will be used.<br>
```
Comment (nir0s) [inline source/guide/3.2/ui-blueprint-upload.md]: Well, first of all, it is mentioned in the beginning of the page that a file named `blueprint.yaml` must exist.
Secondly, how come the blueprint name is a required field if it has a default value?

File: source/guide/3.2/ui-blueprint-upload.md [hunk 4]
```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
 ![The blueprint upload button](/guide/images/ui/ui_upload_blueprint_button.png)
```
Comment (nir0s) [inline source/guide/3.2/ui-blueprint-upload.md]: Also, the sentence: Cloudify's UI allows the user to upload a packaged blueprint TAR file... is written twice. Once in the beginning and once under #uploading a blueprint.

File: source/guide/3.2/ui-blueprint-upload.md (hunk 3 above)
Comment (SefiGiga) [inline source/guide/3.2/ui-blueprint-upload.md]: Nir,

I think you got confused between the blueprint name (which is required) and the blueprint yaml file (which is optional). 

if no file name is provided, the default file name 'blueprint.yaml' file will be used (or expected to exist in the uploaded file).

Maybe 'expected to exist in the uploaded file' is a better way of putting it?

File: source/guide/3.2/ui-login.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+# Login Screen
+After bootstrapping the manager with security support, the UI will redirect the users to the Login screen.<br>
+The standard `Username` and `Password` are required as well as the option to check `Remember me`. <br>
+
+![The Login Screen](/guide/images/ui/ui-login-screen.png)
+
+# Logout
+Once logged in, the user name is displayed on the top right of the screen, right next to the `Log Out` button. <br>
+The user can opt to logout anytime by simply clicking on the `Log Out` (as shown below). <br>
```
Comment (nir0s) [inline source/guide/3.2/ui-login.md]: on the `Log Out` button (as ...)

File: source/guide/3.2/ui-blueprint-upload.md [hunk 6]
```diff
@@ -27,6 +29,11 @@ tar czf blueprint-name.tar.gz blueprint-folder/
 
 The output file of the tar command above will be `blueprint-name.tar.gz`.
 
+Supported file extension are:
+
+* tar
+* tar.gz
+* archive
```
Comment (nir0s) [inline source/guide/3.2/ui-blueprint-upload.md]: what's an archive? everything is an archive.

File: source/guide/3.2/ui-blueprint-upload.md [hunk 7]
```diff
@@ -27,6 +29,11 @@ tar czf blueprint-name.tar.gz blueprint-folder/
 
 The output file of the tar command above will be `blueprint-name.tar.gz`.
 
+Supported file extension are:
```
Comment (nir0s) [inline source/guide/3.2/ui-blueprint-upload.md]: If we support multiple file formats, we need to remove "TAR" from everywhere else..

File: source/guide/3.2/ui-blueprint-upload.md (hunk 6 above)
Comment (SefiGiga) [inline source/guide/3.2/ui-blueprint-upload.md]: Erez confused me. Only tar.gz is supported.

File: source/guide/3.2/ui-blueprint-upload.md (hunk 7 above)
Comment (SefiGiga) [inline source/guide/3.2/ui-blueprint-upload.md]: Erez confused me. Only tar.gz is supported.

File: source/guide/3.2/ui-blueprint-upload.md (hunk 4 above)
Comment (SefiGiga) [inline source/guide/3.2/ui-blueprint-upload.md]: Correct, fixed it.

File: source/guide/3.2/ui-login.md (hunk 5 above)
Comment (SefiGiga) [inline source/guide/3.2/ui-login.md]: Not sure I understand what you want me to do here...
It's already written "on the Log Out button"

**Pull request merged**